apps/stix-shifter/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM debian:buster-slim
+
+RUN apt-get -qq update && apt-get -qqy install \
+  python3-pip software-properties-common
+
+RUN pip3 install --upgrade pip
+
+WORKDIR /opt/tenzir/threatbus/stix-shifter-threatbus
+COPY setup.py .
+COPY README.md .
+COPY stix_shifter_threatbus stix_shifter_threatbus
+RUN python3 -m pip install .
+COPY config.yaml.example config.yaml
```

Comment:
I just want to make sure I understand this correctly: This is step is just used to create *any* file called `config.yaml` so the user can later mount his own config on top when running the container, right? (ie. in principle we could also `RUN touch config.yaml` here?)

Reply:
Alternatively, we could opt for simply printing the helptext via `-h` as default CMD and exit.